### ISSUE TITLE
Option to force reload vocabularies

### ISSUE THREAD

**Issue by MaximMoinat** [thehyve/delphyne#165]
**How would this feature be useful?**
You already have an OMOP vocabulary loaded, but need to load in new one.

**Describe the solution you'd like**
Have an option in the config like `force_reload_vocabularies` that will trigger truncation of existing vocab tables and then load the vocabularies.

Example of a previous (quick-and-dirty) implementation:
https://github.com/EHDEN/ETL-UK-Biobank/blob/8ce0d7e6eb181beb8b57f6d098c432f117c2c81a/src/main/python/initialize_database.py#L42-L62

**Describe alternatives you've considered**
Alternatively, this would be a manual process (as in deleting the vocab schema before running the ETL with loading of standard vocab).

**Comment by Spayralbe** [comment]
I did not add that option, because I thought the risk of forgetting to reset the option to False (after loading has completed) would be rather high. Before you realize your mistake, the vocabularies would already have been dropped and you have to wait quite some time before they're all loaded again.

Therefore I currently prefer the manual deletion step, but if you think we need to have a reload option, we could look at alternatives (maybe manual confirmation?).

**Comment by MaximMoinat** [comment]
Good point. How about doing the truncation and insertion of a vocab table in the same transaction. Such that, if you by accident force reload, then you can cancel (ctrl-c) when you see the reloading happening in the log.

OR: if you set `do_load_standard_vocabulary: True`, then if vocabulary already loaded ask whether it should be reloaded. That is two birds with one stone, because a very minor issue I have now is that the ETL returns with an error if I forget to turn off vocabulary loading when the vocab has been loaded in a previous run.

**Comment by Spayralbe** [comment]
Yes let's go for the second option.
So whenever the vocab tables are not all empty and `do_load_standard_vocabulary` is set to `True`, you will get a console prompt asking you to confirm whether you really want to replace the vocabularies.
